### DISCUSSION TITLE
Button class not working livewire

### DISCUSSION THREAD

**Discussion by webmclink** [filamentphp/filament#9074] [Help]
I followed pretty much every steps here https://filamentphp.com/docs/3.x/forms/installation
but it seems the button class is not working,

![image](https://github.com/filamentphp/filament/assets/60499148/d1077c86-9115-4c69-86f4-b6338d2e4bb9)

it shows white only
![image](https://github.com/filamentphp/filament/assets/60499148/f7528223-ab70-4cd3-b6d3-8fd2e35032f2)

**Comment by sinnbeck** [comment]
Can you share the actual code instead of a screenshot of it? Its hard when we cannot see it all.

**Comment by leandrocfe** [comment]
Are you using Form Builder or Panel Builder?
If you are using Panel builder, you should [create a custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme). This will add the new tailwind classes that Filament isn't using.

**Comment by webmclink** [comment]
I re-installed everything with this command `php artisan filament:install --scaffold --forms` and it working fine now. Thank you for your time @sinnbeck @leandrocfe
